platform/accessibility/guidance/post-launch-audit-processes.md

```diff
@@ -14,7 +14,10 @@ The post-launch audit will focus on manual testing with mobile and assistive tec
 * Browsers such as Edge Chromium, AVG, or Waterfox. Additional software may be tested as time allows. Preferered pairings may change depending on analytics data.
 
 ## Issue Gathering and Triage
-* Issues will be collected as issues in a new ZenHub epic. The accessibility specialist will assign this epic to the person who requested the staging review.
-* Issues will be triaged according to their severity. Some issues will be launch blockers. Others can be fixed post-launch. Teams should review the [accessibility defect severity rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md) if they have questions about issue severity or impact.
-* The accessibility specialist will communciate the application or content's readiness for review to the VA 508 office. The specialist will list known issues and serve as a liaison between the requesting team and the VA 508 representative.
-* Accessibility specialists will work with stakeholders as issues are ready to be re-tested. When an issue has been resolved, the specialist or project manager should close it with a brief comment like _"Issue has been tested, and success criteria met. Moving to close."_
+* Issues will be collected as issues in a new ZenHub epic. The accessibility specialist will assign this epic to the project manager (PM) after the audit is complete.
```

Comment:
I think you mean product manager here

Reply:
Actually you mention Project Manager many places shere I think you mean product

Reply:
Did a full text find and replace, got all of the instances changed to product managers.